packages/components/src/Button/Button.css

```diff
@@ -16,22 +16,24 @@
   border: var(--border-base) solid var(--button--color-variation);
   border-radius: var(--radius-base);
   color: var(--button--color-variation);
-  fill: var(--button--color-variation);
   -webkit-font-smoothing: antialiased;
+  letter-spacing: 0.2px;
   text-align: center;
   text-decoration: none;
   vertical-align: middle;
   background-color: var(--button--color-surface);
   cursor: pointer;
   transition: all var(--timing-base);
   appearance: none;
+  fill: var(--button--color-variation);
```

Comment:
Did this move itself on save (auto-formattting?), or was this a specific decision?

Reply:
@chris-at-jobber yes, this moved itself. I noticed there's quite a few things my vsCode changes which go against the Atlantis linter, so I'll be making sure those settings are adjusted